client/state/jetpack-connect/actions/create-account.js

```diff
@@ -27,6 +27,8 @@ export function createAccount( userData ) {
 				client_id: config( 'wpcom_signup_id' ),
 				client_secret: config( 'wpcom_signup_key' ),
 			} );
```

Comment:
It might be worth checking user data to see if there are any fields that can be used to customize the email template.